test/fixtures/python/sum.invalid.py

```diff
@@ -0,0 +1,10 @@
+num = eval(input("Enter a positive number: "
```

Comment:
The syntax errors here are largely the same as the other example.  Why do we need two examples?  I would understand having one for Python and one for Go.  But two for Python that have missing ( or ) doesn't give much value.

Also, you're missing the copyright and SPDX

Reply:
I wanted to see how will look the output when we have two bad files.